client/a8c-for-agencies/sections/referrals/lib/get-estimated-commission.ts

```diff
@@ -0,0 +1,86 @@
+import { APIProductFamilyProduct } from '../../../../state/partner-portal/types';
+import { Referral } from '../types';
+import { getNextPayoutDateActivityWindow } from './get-next-payout-date';
+
+export const getProductCommissionPercentage = ( slug?: string ) => {
```

Comment:
Looks like the same function is defined [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/a8c-for-agencies/sections/referrals/lib/commissions.ts#L4) if we want to reuse it and avoid duplication.

Reply:
Ah, yes, I plan to remove that file afterwards, so will just keep it duplicated for now.